schemas/EGA.analysis.json

```diff
@@ -5,9 +5,190 @@
+        ]
+      },
+
+      "schema_descriptor": {
+        "title": "Schema descriptor node",
+        "description": "Inherited schema descriptor node containing metadata about the schemas and standards used to create the JSON document itself.",
+        "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/schema_descriptor"
+      },
+
+      "object_title": {
+        "type": "string",
+        "title": "Title of the analysis",
+        "description": "An informative analysis title that should serve as an overview of the analysis, including: performed analysis, samples, purpose... (e.g. 'Variant calling analysis of tumor repressed cells'). This short text can be used to call out analyses records in searches or in displays.",
+        "examples": [ "Variant calling analysis of tumor repressed cells" ]
+      },
+
+      "object_description": {
+        "type": "string",
+        "title": "Description of the analysis",
+        "description": "An in-depth description of the biological relevance and intent of the analysis, including its workflow.",
+        "examples": [ "The analysis was conducted with the objective of... ...and for that purpose we compared untreated controls against..." ]
+      },
+
+      "sample_number": {
+        "type": "integer",
+        "title": "Number of samples of the analysis",
+        "description": "Number of samples included in the analysis (i.e. analysed together). One sample corresponds to one biological replicate [EFO:0002091] (e.g. genetic content from a single cell, a tissue, buccal swab, etc.) from a single individual or from several individuals. Shall not be mistaken for technical replicates [EFO:0002090] being used several times (see https://www.ebi.ac.uk/seqdb/confluence/pages/viewpage.action?spaceKey=EGA&title=Sample+Representation).",
+        "examples": [ 30 ]
+      },
+
+      "analysis_protocols": {
+        "type": "array",
+        "title": "Protocols related to an analysis",
+          "description": "Comprises metadata (e.g. Type of protocol) of a plan specification executed during an analysis. It shall have a sufficient level of detail and quantitative information to communicate it (and thus reproduce it) between investigation agents.",
+        "minItems": 1,
+        "additionalProperties": false,
+        "uniqueItems": true,
+        "items": {
+          "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/protocols_object" 
+        }
+      },
+
+      "targeted_loci": {
+        "type": "array",
+        "title": "Loci of the targeted genomic feature",
+        "description": "Array of items that unambiguously define the loci of targeted genomic features in the analysis. For example, if the aim of the analysis was to detect variants in the human gene TAF1 and TP53, their identifiers will be expected in two items of this array.",
+        "minItems": 1,
```

Comment:
```suggestion
```

Reply:
This attribute must not be required. Some analyses, e.g. whole genome sequencing to detect genomic variation, so not have targeted genomic features.